java/CustomMetrics/README.md

```diff
@@ -3,18 +3,34 @@
 * Flink version: 1.20
 * Flink API: DataStream API
 * Language: Java (11)
+* Flink connectors: Kinesis Sink
```

Comment:
if it uses Datagen connector, should we specify? or only if it requires dependency

Reply:
I was thinking to omit it not to create confusion.
It's formally a connector, but not really connecting to anything.